manifest/spark-operator-with-webhook.yaml

```diff
@@ -78,7 +79,7 @@ spec:
       - name: main
         image: gcr.io/spark-operator/spark-operator:v2.4.0-v1beta1-latest
         imagePullPolicy: IfNotPresent
-        command: ["/usr/bin/gencerts.sh", "-p"]
+        command: ["/usr/bin/gencerts.sh", "-n", "spark-operator", "-s", "spark-webhook", "-p"]
```

Comment:
Do we need this change?

Reply:
You are right. I forgot that they are already defaults in `gencerts.sh`. I've undo this change.